playbooks/repec.yml

```diff
@@ -1,9 +1,12 @@
 ---
-- hosts: repec_prod
+# by default this playbook runs in the staging environment
+# to run in production, pass '-e runtime_env=production'
+- hosts: repec_{{ runtime_env | default('staging') }}
   remote_user: pulsys
   become: true
   vars_files:
-    - ../group_vars/repec/repec_prod.yml
+    - ../group_vars/repec/repec_{{ runtime_env | default('staging') }}.yml
+    - ../group_vars/repec/common.yml
     - ../group_vars/repec/vault.yml
```

Comment:
Is there something that is supposed to happen with the assets repository or something after you run this playbook? Would that be good to add to the post_tasks?

Reply:
Bundler will handle pulling in the pul_assets repo.  The only thing to do is deploy the app, which doesn't seem worth a special `post_tasks` reminder to me.